pricepoint_intel/api/ingestion_routes.py

```
@@ -0,0 +1,811 @@
+            "success": True,
+            "message": "Database initialized successfully",
+            "tables_created": [
+                "skus",
+                "vendors",
+                "vendor_pricing",
+                "geographic_markets",
+                "distribution_centers",
+                "price_history",
+            ],
+        }
+    except Exception as e:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)